recipes/ablog/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "ablog" %}
+{% set version = "0.9.1" %}
+{% set sha256 = "d8d3a347b617a87155d9a32d51d82ad1d5e4ca1944620fbde54be18137406b39" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - ablog = ablog.commands:ablog_main
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - werkzeug
+    - sphinx
+    - alabaster
+    - invoke
+    - python-dateutil
+    - sphinx-automodapi
+
+test:
+  commands:
+    - ablog --version
```

Comment:
I am wondering if we should make this build the docs locally to check that it runs correclty

Reply:
Sure, if this is simple and doesn't introduce other dependencies.

Reply:
I think, but would have to check the CI after but:

```
	- cd docs
        - ablog build -T
	- ablog build -b latex -T -d .doctrees -w _latex
	- ablog build -T -b json
	- ablog build -T -b pickle
```

Should be enough. It might need the `cd` be more explicit, let us see what the CI tells us.